robotframework-ls/tests/robotframework_ls_tests/test_semantic_highlighting_bdd.py

```diff
@@ -0,0 +1,199 @@
+from robotframework_ls.impl.completion_context import CompletionContext
```

Comment:
All the imports should be made locally and not globally in this module.

Reply:
Fixed